StripeUICore/StripeUICore/Source/Validators/PhoneNumber.swift

```diff
@@ -343,7 +343,6 @@ import UIKit
             Metadata(prefix: "+509", regionCode: "HT", pattern: "## ## ####"),
             Metadata(prefix: "+51", regionCode: "PE", pattern: "### ### ###"),
             Metadata(prefix: "+52", regionCode: "MX", pattern: "### ### ####"),
-            Metadata(prefix: "+537", regionCode: "CY", pattern: ""),
```

Comment:
See the existing line that correctly handles this here: https://github.com/stripe/stripe-ios/blob/master/StripeUICore/StripeUICore/Source/Validators/PhoneNumber.swift#L292